src/Wallabag/CoreBundle/Form/Type/EditEntryType.php

```diff
@@ -22,6 +22,11 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'required' => false,
                 'label' => 'entry.edit.url_label',
             ])
+            ->add('origin_url', TextType::class, [
```

Comment:
Maybe we should use `UrlType` instead (and same on line 16).
I don't know why we use text instead of url 🤔